Fixes an issue in yaml-cpp v0.7.0 where the static libraries could not be found
for linking

https://github.com/jbeder/yaml-cpp/pull/1037

commit 4f48727b365962e31451cd91027bd797bc7d2ee7
Author: Artturin <Artturin@artturin.com>
Date:   Tue Sep 28 00:11:04 2021 +0300

    yaml-cpp-config.cmake.in: apply patch to fix #774
    
    From https://github.com/jbeder/yaml-cpp/issues/774#issuecomment-909274732

diff --git a/yaml-cpp-config.cmake.in b/yaml-cpp-config.cmake.in
index 7b41e3f..de0981d 100644
--- a/yaml-cpp-config.cmake.in
+++ b/yaml-cpp-config.cmake.in
@@ -5,10 +5,10 @@
 
 # Compute paths
 get_filename_component(YAML_CPP_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-set(YAML_CPP_INCLUDE_DIR "@CONFIG_INCLUDE_DIRS@")
 
 # Our library dependencies (contains definitions for IMPORTED targets)
 include("${YAML_CPP_CMAKE_DIR}/yaml-cpp-targets.cmake")
 
 # These are IMPORTED targets created by yaml-cpp-targets.cmake
-set(YAML_CPP_LIBRARIES "@EXPORT_TARGETS@")
+get_target_property(YAML_CPP_INCLUDE_DIR yaml-cpp INTERFACE_INCLUDE_DIRECTORIES)
+get_target_property(YAML_CPP_LIBRARIES yaml-cpp IMPORTED_LOCATION_RELEASE)
